po/plume-front/en.po~

```diff
@@ -0,0 +1,45 @@
+msgid ""
```

Comment:
aren't ~ files backup some editor make?

Reply:
These files are generated by the gettext command line tools, but they are normally gitignored. I'll remove them.